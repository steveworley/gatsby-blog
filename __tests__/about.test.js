import React from 'react'
import renderer from 'react-test-renderer'
import About from '../src/pages/about'

jest.mock('../src/components/layout',  () => 'Layout')

const data = {
  data: {
    site: {
      siteMetadata: {
        title: 'Testing'
      }
    }
  }
}

describe("About", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<About {...data} />).toJSON()
    expect(tree).toMatchSnapshot();
  })
})