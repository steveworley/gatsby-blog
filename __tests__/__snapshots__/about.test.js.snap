// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
<Layout>
  <h1>
    About 
    Testing
  </h1>
  <p>
    We're the only site running on your computer dedicated to showing the best photos and videos of pandas eating lots of food.
  </p>
</Layout>
`;
